Automattic/wp-calypso
# Add A/B test for Professional Email as plan feature on logged in plans page

#### Changes proposed in this Pull Request

* This PR implements the copy changes and A/B test to have Professional Email be listed as a plan feature on the logged-in plans page at `/plans/:siteSlug`
* The new Professional Email feature shows for both annual and monthly plans
* If we have a situation where we're loading the experiment assignment for a user, we show a grey box for a placeholder.

#### Testing instructions

* Visit the experiment page for this A/B test, and ensure that your user is assigned to the `treatment` for the experiment
* Run this branch locally or via [the Calypso live branch](https://github.com/Automattic/wp-calypso/pull/61149#issuecomment-1041193069)
* Navigate to **Upgrades** -> **Plans** while logged in
* While displaying annual plans, verify that you see the new feature listed for all paid plans, and that you can get additional information via the info icon
* Switch to view the monthly plans, and verify that the new feature is listed for all paid plans, and that you can get additional information via the info icon
* Access your browser developer tools and remove the Local Storage data with the key `explat-experiment--calypso_promote_professional_email_as_a_plan_feature_2022_02`.
* Visit the experiment page for this A/B test, and ensure that your user is assigned to the `control` for the experiment
* Navigate to **Upgrades** -> **Plans** while logged in
* Verify that the Professional Email feature is not listed for monthly or annual plans

#### Screenshots

##### Free trials promoted for plans

<img width="1712" alt="Free Professional Email feature" src="https://user-images.githubusercontent.com/3376401/154216123-0fe8e3d9-4221-4554-b950-2f85d218b01a.png">

##### Free trials promoted for plans with hover text displayed

<img width="1710" alt="Hover text for free Professional Email feature" src="https://user-images.githubusercontent.com/3376401/156425718-d7238a88-ef6f-484a-aa21-d4c36660e92d.png">
